src/test/java/com/google/api/codegen/testsrc/libraryproto/library_gapic.yaml

```diff
@@ -240,9 +240,9 @@ interfaces:
         - edition=123
         attributes:
         - parameter: shelf.name
-          sample_argument: true
+          sample_argument_name: name
```

Comment:
I think we could s/`sample_argument_name`/`sample_argument`/ , iejust changing the type of the field